lab_1/Laboratory Work #1 Liya Kunchenko - 23FPL1

```diff
@@ -0,0 +1,9 @@
+Lab 1.
+"""
+
+
+def say_hello_world() -> None:
+    print('Hello world!')
+
+
+say_hello_world()
```

Comment:
Revert this. You must have only 1 directory with lab and it's name is "lab_1_classify_by_unigrams". Artyom Mikhailovich will help you at the next seminar.

Reply:
?